src/services/gapi.js

```diff
@@ -0,0 +1,36 @@
+/* global gapi */
+import $S from 'scriptjs';
+import once from 'lodash/once';
+import config from '../config';
+
+export const loadGapi = once(async() => new Promise((resolve) => {
```

Comment:
Is there a reason to export this function?

Reply:
nope - I think I miss understood how it was working originally